### PR TITLE
(DOCS): Edit Youtube Broadcasting

### PR DESCRIPTION
**Pull request by paulofariadias**

### CODE REVIEW

File: administrator-guides/youtube-broadcasting/README.md [hunk 1]
```diff
@@ -1,20 +1,21 @@
...
-> Please note that this feature will only be available from versions 0.66 and forward.
+This feature adds an integration between Rocket.Chat and Youtube Live. When enabled and configured, your channel owners can broadcast their camera feed live to the Livestream feature, which allows all users on the channel to watch a common stream.
 
-This feature adds an integration between Rocket.Chat and Youtube Live, when enabled and set up, your channel owners will be able to broadcast their camera feed live to the Livestream feature which allows all users on the channel to watch a common stream.
+**Note**: This feature is only available from versions 0.66 and above.
 
 ## Configuration
 
-The Broadcasting feature can only be used with the Livestream Tab feature.
+The `Broadcasting` feature can only be used with the `Livestream Tab` feature.
```
Comment (cherwoo) [inline administrator-guides/youtube-broadcasting/README.md]: The Broadcasting feature is available on the Livestream tab.

File: administrator-guides/youtube-broadcasting/README.md [hunk 2]
```diff
@@ -1,20 +1,21 @@
-# Youtube Broadcasting
+# Integration with Youtube Live
 
-> Please note that this feature will only be available from versions 0.66 and forward.
+This feature adds an integration between Rocket.Chat and Youtube Live. When enabled and configured, your channel owners can broadcast their camera feed live to the Livestream feature, which allows all users on the channel to watch a common stream.
```
[the remark this answers is not in the document]
Comment (paulofariadias) [reply administrator-guides/youtube-broadcasting/README.md]: Changes implemented.

File: administrator-guides/youtube-broadcasting/README.md [hunk 3]
```diff
@@ -1,20 +1,21 @@
...
 ## Configuration
 
-The Broadcasting feature can only be used with the Livestream Tab feature.
+The `Broadcasting` feature can only be used with the `Livestream Tab` feature.
 
-You can find the settings for this feature on `Livestream & Broadcasting` in the admin panel.
+Find the configuration for this feature in `Livestream & Broadcasting`, in the Admin panel.
 
-If you set the `Broadcasting Enable` setting to false, users wont be able to create live streams using their webcams, but will still be able to link a live stream from youtube.
+If you set `Broadcasting Enable` to false, users are unable to create live streams using their webcams, but they can still add a link to a live stream from Youtube.
```
[the remark this answers is not in the document]
Comment (paulofariadias) [reply administrator-guides/youtube-broadcasting/README.md]: Implemented.

File: administrator-guides/youtube-broadcasting/README.md [hunk 4]
```diff
@@ -1,20 +1,21 @@
...
+**Note**: Currently, the Admin needs a valid Google app with access to the Youtube API to use Youtube live integration. Find more details on the [Google Developer page](https://developers.google.com/youtube/v3/)). This allows the administrator to have a valid API Key, Client ID & Secret, which are used on the Broadcasting feature.
 
-## Livestream tab
+## Livestream Tab
 
-After a correct set up, the Livestream tab will now allow you to Broadcast your camera when editing the source url.
-When opening the popout, it might take a moment while we prepare the broadcast and livestream for your session, this will be indicated by the blinking icon and "Preparing" status displayed in the control bar. After that initial phase, the broadcast will be Ready and available to start with your click.
+After proper configuration, the `Livestream tab` allows to Broadcast your camera when your edit the source URL.
```
[the remark this answers is not in the document]
Comment (paulofariadias) [reply administrator-guides/youtube-broadcasting/README.md]: Implemented.

File: administrator-guides/youtube-broadcasting/README.md [hunk 5]
```diff
@@ -1,20 +1,21 @@
...
+
+When your open the pop-out window, it may take a moment while we prepare the broadcast and livestream for your session. The blinking icon, and the `Preparing` status displayed in the control bar, indicate this preparation. After that, your broadcast is `Ready` and available to start with your click.
```
[the remark this answers is not in the document]
Comment (paulofariadias) [reply administrator-guides/youtube-broadcasting/README.md]: Implemented.

File: administrator-guides/youtube-broadcasting/README.md (hunk 1 above)
Comment (paulofariadias) [reply administrator-guides/youtube-broadcasting/README.md]: Implemented.

File: administrator-guides/youtube-broadcasting/README.md [hunk 6]
```diff
@@ -1,20 +1,21 @@
...
 
-You can find the settings for this feature on `Livestream & Broadcasting` in the admin panel.
+Find the configuration for this feature in `Livestream & Broadcasting`, in the Admin panel.
 
-If you set the `Broadcasting Enable` setting to false, users wont be able to create live streams using their webcams, but will still be able to link a live stream from youtube.
+If you set `Broadcasting Enable` to false, users are unable to create live streams using their webcams, but they can still add a link to a live stream from Youtube.
 
-_Currently the admin needs a valid Google app with access to the Youtube API to use youtube live integration (More details on the [Google Developer page](https://developers.google.com/youtube/v3/)), this will allow the administrator to have a valid API Key, Client ID & Secret which are used on the Broadcasting feature._
+**Note**: Currently, the Admin needs a valid Google app with access to the Youtube API to use Youtube live integration. Find more details on the [Google Developer page](https://developers.google.com/youtube/v3/)). This allows the administrator to have a valid API Key, Client ID & Secret, which are used on the Broadcasting feature.
```
[the remark this answers is not in the document]
Comment (paulofariadias) [reply administrator-guides/youtube-broadcasting/README.md]: Implemented.
Martin says "This" refers to having access to YouTube APIs.

**Pull request closed without merging**